utils/datautils.js

```diff
@@ -108,25 +108,8 @@ function getLeaderboard(options) {
 function getCounts() {
     const participants = db.Claim.aggregate('user', 'count', {distinct: true});
     const claims = db.Claim.aggregate('*', 'count');
-    var accepted = db.Claim.aggregate(
-        'bounty',
-        'sum',
-        {
-            where: {
-                "status": "accepted"
-            }
-        }
-    );
-    var totalclaimed = db.Claim.aggregate(
-        'bounty',
-        'sum'
-    );
-    if (Number.isNaN(accepted)) {
-        accepted = 0;
-    }
-    if (Number.isNaN(totalclaimed)) {
-        totalclaimed = 0;
-    }
+    const accepted = db.Database.query(`SELECT coalesce(sum("bounty"),0) AS "sum" FROM "claims" AS "claim" WHERE "claim"."status" = 'accepted'`,{ type: db.Database.QueryTypes.SELECT });
```

Comment:
Let's keep ORM query for its readability over raw database query. I'm sure we can check for NaN/falsy values after this